src/quantum/tp6.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import { frum } from '../utils/query_ops';
+import { TP6_PAGES } from './config';
+import DashboardPage from '../components/DashboardPage/index';
+import PerfherderGraphContainer from '../containers/PerfherderGraphContainer/index';
+
+const styles = {
+  body: {
+    backgroundColor: 'white',
+  },
+  chart: {
+    justifyContent: 'center',
+    padding: '1rem',
+  },
+};
+
+class TP6 extends React.Component {
+  render() {
+    const { classes, location } = this.props;
+    const params = new URLSearchParams(location.search);
+    const limits = { bits: params.get('bits') };
```

Comment:
This will be replaced with `props` eventually, ounce we have a `<Navigation>` like firox-perf-dashboard https://github.com/mozilla-frontend-infra/firefox-performance-dashboard/blob/master/src/App/index.jsx#L34